assignments/objects.js

```diff
@@ -17,32 +17,77 @@ let example = {
   "gender": "F"
 }
 
+
 // Write your intern objects here:
+let mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmellow0@psu.edu",
+  "gender": "F"
+}
 
+let kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
 
-// ==== Challenge 2: Reading Object Data ==== 
+let keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmurrery2@wikimedia.org",
+  "gender": "M"
+}
+let gannie = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gratinson3@illinois.edu",
+  "gender": "M"
+}
+let antonietta = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(mitzi.name);
 // Kennan's ID
-
+console.log(kennan.id);
 // Keven's email
-
+console.log(keven.email);
 // Gannie's name
-
+console.log(gannie.name);
 // Antonietta's Gender
+console.log(antonietta.gender);
 
-// ==== Challenge 3: Object Methods ==== 
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+
+
+function speak(object){
```

Comment:
So here we want to give the `kennan` object a method called `speak`. You can do this just like you would assign any value to a variable, only instead of a string or number, it will be a function.

```
kennan.speak = function speak() {
    return 'Hello my name is Kennan.';
}
```

I really like how you thought about this though, passing the object in as an argument and then pulling the name from the object!